packages/trimerge-sync-basic-server/src/lib/SqliteDocStore.ts

```diff
@@ -128,18 +102,15 @@ export class SqliteDocStore implements DocStore {
     const remoteSyncId = this.syncIdCreator();
     const insert = this.db.prepare(
       `
-        INSERT INTO commits (ref, remoteSyncId, remoteSyncIndex, userId, baseRef, mergeRef, mergeBaseRef, delta, metadata, main) 
-        VALUES (@ref, @remoteSyncId, @remoteSyncIndex, @userId, @baseRef, @mergeRef, @mergeBaseRef, @delta, @metadata, @main)
+        INSERT INTO commits (ref, remoteSyncId, remoteSyncIndex, userId, baseRef, mergeRef, delta, metadata) 
```

Comment:
I wonder if it makes sense for us to keep main on the server but implement it as a userland construct?

Reply:
resolved this, the main bool is now stored in metadata, though